src/libraries/Common/src/Roslyn/GlobalAliases.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.DotnetRuntime.Extensions;
```

Comment:
NIT: Not sure if it's part of our coding style or not, but I see that for some files you are using this new namespace declaration, while for other files you are using the namespace block, it would be good to unify.

Reply:
I do not think style/unification should be considered for this pr at all. An overarching goal is to keep this in sync with Roslyn. Unnecessary style tweaks will just make that harder. We should really treat this pr as a hack where a chunk of roslyn is source linked in in the interim period since the sdk cannot reference this new API directly. 

Once the new API can be used based on your release process, you can jettison all these files.